ballerina/http_client.bal

```diff
@@ -49,7 +49,7 @@ client isolated class HttpClient {
         return externExecuteClientAction(self, path, <Request>message, HTTP_HEAD);
     }
 
-    # The `HttpClient.put()` function can be used to send HTTP PUT requests to HTTP endpoints.
+    # Update or replace existing data at an HTTP endpoint. Can be used to modify complete resources in web APIs.
```

Comment:
Update is mostly done by PATCH. PUT can also create resources. The difference between POST and PUT, is that PUT is idempotent.

```suggestion
    # Create a new resource or replace a representation of the target resource
```